.circleci/config.yml

```diff
@@ -401,29 +410,23 @@ jobs:
           command: echo "Test modules succeeded"
 
   heartbeat_demo:
-    <<: *python_36_base
+    <<: *python_37_base
```

Comment:
Just curious: why python 3.7 here?

Reply:
`breakpoint()`? :-)

Reply:
isn't 3.7 our target version in general?
Why 3.6?

Reply:
We dont have a single target version to build for, but we're trying to support 3.6+ - I think most folks contributing to nucypher are using 3.7 now.